src/test/java/carRacingWinner/entity/Car.java

```diff
@@ -1,16 +1,30 @@
 package carRacingWinner.entity;
 
+import java.util.Random;
+
 public class Car {
 
     String name;
+    private final String display = "-";
```

Comment:
요거는 인스턴스 변수라기보단, 상수에 가까운 것 같습니다. 상수의 네이밍 콘벤션(대문자 스네이크 케이스), 그리고 `static final` 키워드를 사용해 주시면 좋을 것 같습니다.

```suggestion
    private static final String DISPLAY = "-";
```

그리고.. 무엇을 위한 디스플레이인지, 변수명 자체도 좀 더 분명하면 좋을 것 같네요.